### PR TITLE
fix(ajax): call ajax take time with big db. Limit to 10 should be good enough

### PR DESCRIPTION
**Pull request by chadyred**
**WHY ?**

With lots of email, it is slow to load data.

**HOW ?**

Add a parameter with limit to 10. For autocomplete, it is good enough.

### CODE REVIEW

**Comment by chadyred** [comment]
@loevgaard CI not pass for other reason.

**Comment by loevgaard** [comment]
Hi @chadyred 

Thanks for your PR. Could you just rebase your branch onto master? I fixed the build

File: src/Doctrine/ORM/CustomerRepositoryTrait.php [hunk 1]
```diff
@@ -16,11 +16,12 @@ trait CustomerRepositoryTrait
      */
     abstract public function createQueryBuilder($alias, $indexBy = null);
 
-    public function findByEmailPart(string $email): array
+    public function findByEmailPart(string $email, int $limit): array
```
Comment (loevgaard) [inline src/Doctrine/ORM/CustomerRepositoryTrait.php:19]:
```suggestion
    public function findByEmailPart(string $email, int $limit = 10): array
```

File: src/Repository/CustomerRepositoryInterface.php [hunk 2]
```diff
@@ -8,5 +8,5 @@
 
 interface CustomerRepositoryInterface extends BaseCustomerRepositoryInterface
 {
-    public function findByEmailPart(string $email): array;
+    public function findByEmailPart(string $email, int $limit): array;
```
Comment (loevgaard) [inline src/Repository/CustomerRepositoryInterface.php:11]:
```suggestion
    public function findByEmailPart(string $email, int $limit = 10): array;
```

File: src/Resources/config/routes/admin_ajax.yaml [hunk 3]
```diff
@@ -11,6 +11,7 @@ setono_sylius_gift_card_admin_ajax_customer_by_email_phrase:
                 method: findByEmailPart
                 arguments:
                     email: $phrase
+                    limit: 10
```
Comment (loevgaard) [inline src/Resources/config/routes/admin_ajax.yaml:14]: You can revert this change :)

**Comment by chadyred** [comment]
Done !

**Comment by loevgaard** [comment]
Released as 0.8.5 :) 

Thank you!
